doc/src/content/docs/installing/2-new-versions.mdx

```diff
@@ -2,8 +2,11 @@
 title: Updating to new versions
 ---
 import { Steps } from '@astrojs/starlight/components';
+import { Aside } from '@astrojs/starlight/components';
 
-
+<Aside type="tip">
+    As a developer, you can use the script `./update.sh` to update the application. Note that local changes will be lost.
```

Comment:
@ragusa87 When I read this, I will never use this script... I know what you mean but it should be something like 
```suggestion
    As a developer, you can use the script `./update.sh` to update the application. If you have local changes in the application code, they will be overwritten by the latest release. 
```